.gitignore

```diff
@@ -0,0 +1,6 @@
+query.log
```

Comment:
That's my mistake I left it out of the first pull request. Here you go:

https://github.com/ArpNetworking/arpnetworking-parent-pom/pull/2

I was thinking it would just be easier to have one .gitignore that can be copied from any project to any other since we'll be making so many repositories in the next couple of weeks. Thoughts?